avocado/formatters.py

```diff
@@ -287,7 +295,7 @@ def get_meta(self, exporter=None):
 
         return meta
 
-    def to_string(self, value, **context):
+    def to_string(self, value, field, context):
```

Comment:
The purpose of `context` is contain things like `request` or `user` or anything else that could be passed in various contexts. The arguments being passed in are [fixed](https://github.com/chop-dbhi/avocado/pull/314/files#diff-6dc2a57dc20e90ba6a3c899398b29820R156) so these base methods need to take `context` even if they aren't being used in scope.